sdk/messaging/azservicebus/internal/amqpwrap/amqpwrap_test.go

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package amqpwrap
```

Comment:
Empty file?

Reply:
Will remove (ended up testing this in a live scenario instead)

Reply:
Deleted.